### PR TITLE
Add workflow to automate running sample apps and add badges

### PR DESCRIPTION
**Pull request by qianz2**
Added a workflow that send a dispatch event to sample apps repo when a push or pull request event occurs. 
Added badges of sample apps showing the status.

### CODE REVIEW

File: .github/workflows/automate-sample-apps.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Send a dispatch event
+      run: |
+          curl -X POST https://api.github.com/repos/MicrosoftOryx/githubactions-samples/dispatches -H 'Accept: application/vnd.github.everest-preview+json' -H 'Authorization: token ${{secrets.REPO_ACCESS_TOKEN}}' --data '{"event_type": "build_application"}'
```
Comment (kichalla) [inline .github/workflows/automate-sample-apps.yml]: To make it easy for anyone of us (collaborators) to kick off a build on-demand, I think having this line in the Readme of the samples repo would be convenient.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # GitHub Action for building Azure Web Apps
 
+![V1_BuildAndDeploy_PythonApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_PythonApp_ToAzureWebApp/badge.svg?branch=master)
+![V1_BuildAndDeploy_PhpApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_PhpApp_ToAzureWebApp/badge.svg?branch=master)
+![V1_BuildAndDeploy_NodeApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_NodeApp_ToAzureWebApp/badge.svg?branch=master)
+![V1_BuildAndDeploy_AspNetCoreApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V1_BuildAndDeploy_AspNetCoreApp_ToAzureWebApp/badge.svg?branch=master)
+![V2Beta_BuildAndDeploy_PythonApp_ToAzureWebApp](https://github.com/MicrosoftOryx/githubactions-samples/workflows/V2Beta_BuildAndDeploy_PythonApp_ToAzureWebApp/badge.svg?branch=master)
```
Comment (kichalla) [inline README.md:7]: All the v2-beta ones should be based on `v2-beta` branch?

**Review by kichalla — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (qianz2) [reply README.md:7]: These is master branch under githubaction-samples repo I think.

File: .github/workflows/automate-sample-apps.yml (hunk 1 above)
Comment (qianz2) [reply .github/workflows/automate-sample-apps.yml]: That makes sense. I'll add one there either. Thanks!

File: README.md (hunk 2 above)
Comment (kichalla) [reply README.md:7]: You are right. Thanks.

Comment (qianz2) [reply README.md:7]: I'm also thinking if there's a way to separate v1 and v2-beta maybe? right now, it'll send dispatch event to all sample apps not matter which branch it updated. If we could send dispatch event to sample apps according to v1 or v2-beta branch separately, that makes sense better.

File: .github/workflows/automate-sample-apps.yml (hunk 1 above)
Comment (kichalla) [reply .github/workflows/automate-sample-apps.yml]: So we do not want this file anymore here right?

File: .github/workflows/automate-sample-apps.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Trigger a repository dispatch event to sample apps repo to run apps
```
Comment (kichalla) [inline .github/workflows/automate-sample-apps.yml]: As discussed offline, remove this file. We do not want to use personal access tokens (even as secrets).

**Review by kichalla — APPROVED** [review]

**Pull request merged**